keyvault/2016-10-01/examples/storage/deleteStorageSasDefinition.json

```diff
@@ -0,0 +1,32 @@
+{
+   "parameters":{
+      "storage-account-name":"deletesas01",
+      "name":"deleteStrgSasDef01",
```

Comment:
think this should be `sas-definition-name` instead of `name`, correct? Same for `GetSasDefinition`, `SetSasDefinition` and `UpdateSasDefinition`